docs/core/build/distribution-packaging.md

```diff
@@ -27,17 +27,30 @@ When installed, .NET Core consists of several components that are laid out as fo
 ├── sdk
```

Comment:
preivew 8 has added a `templates` directory next to the sdk: https://github.com/dotnet/core/issues/3184#issuecomment-521702831

Reply:
@dagood, from our source-build it seems templates are under a runtime versioned folder:
```
$ ls /usr/lib64/dotnet/templates/
3.0.0-preview8-013656
```
Why is it like this, and not the sdk version?

What package will the templates be part of? `dotnet-sdk-[major].[minor]` or their own package (`dotnet-templates-[major].[minor]`?).

Reply:
> What package will the templates be part of? `dotnet-sdk-[major].[minor]` or their own package (`dotnet-templates-[major].[minor]`?).

Peeking at an SDK Deb package from the link table at https://github.com/dotnet/core-sdk, it's in the SDK package for 3.0 preview 9.

It would make sense to me, though, to put them in a different package so users can choose different major-minor versions of templates without installing multiple SDKs. I *think* this is a general direction we want to take for the Microsoft build too, @leecow correct me if I'm wrong.

Reply:
> @tmds I asked the same question about the funny version number here: dotnet/cli#12275

It doesn't make sense for me that there is a runtime version there because it's content that you may want to change with an sdk version, without there being a the runtime version change.

Anyway, the general rule is: if something is under a folder named `[major].[minor].[patch]-[suffix]` it ends up in a package named `packagename-[major].[minor]`.

If we can use the same version for all the packages (cfr https://github.com/dotnet/docs/pull/13196#discussion_r317047143), the package maintainer doesn't need to worry about propagating the `[patch]-[suffix]` into the package versions.

Reply:
I have added a `dotnet-templates-[major].[minor]` package.